src/include/execution/sql/value.h

```diff
@@ -427,7 +341,8 @@ struct ValUtil {
         return static_cast<uint32_t>(common::MathUtil::AlignTo(sizeof(TimestampVal), 8));
       case type::TypeId::DECIMAL:
         // TODO(Amadou): We only support reals for now. Switch to Decimal once it's implemented
-        return static_cast<uint32_t>(common::MathUtil::AlignTo(sizeof(Real), 8));
+        // TODO(WAN): switching to DecimalVal, but we don't have a Real type?
```

Comment:
Nomenclature is inconsistent. Technically `DecimalVal` is a `RealVal` since SQL `decimal` and `numeric` supposed to be fixed precision. Basically everything with a decimal point is inconsistent between the frontend and backend.